portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/Sites.testcase

```diff
@@ -311,6 +312,80 @@ definition {
 			parentSiteName = "Site Name");
 	}
 
+	@description = "This is a test for LPS-102711. The changes of site scope configuration in Instance Settings should be applied to the site."
+	@priority = "4"
+	test ViewChangesOfSiteScopeInInstanceSettings {
+		task ("Add a site via JSON") {
+			JSONGroup.addGroup(groupName = "Test Site Name");
+		}
+
+		task ("Navigate to Site Settings admin in custom site") {
+			ApplicationsMenu.gotoSite(site = "Test Site Name");
+
+			ProductMenu.gotoPortlet(
+				category = "Configuration",
+				portlet = "Site Settings");
+		}
+
+		task ("Navigate to Message Boards") {
+			Click(
+				key_itemName = "Message Boards",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+		}
+
+		task ("View the Checkbox of Message Boards is uncheked") {
+			AssertElementPresent(
+				checkboxName = "Enable Message Boards Moderation",
+				locator1 = "Checkbox#ANY_CHECKBOX_UNCHECKED");
+		}
+
+		task ("Navigate to Applications Menu and Instance Settings") {
+			ApplicationsMenu.gotoPortlet(
```

Comment:
Should access to Instance Settings admin via URL
PortalSettings.openInstanceSettingsAdmin